rclcpp_lifecycle/src/lifecycle_node.cpp

```diff
@@ -81,6 +82,15 @@ LifecycleNode::LifecycleNode(
       options.start_parameter_event_publisher(),
       options.parameter_event_qos_profile()
     )),
+  node_time_source_(new rclcpp::node_interfaces::NodeTimeSource(
+    node_base_,
```

Comment:
```
--- src/lifecycle_node.cpp
+++ src/lifecycle_node.cpp.uncrustify
@@ -86,8 +86,8 @@
-    node_base_,
-    node_topics_,
-    node_graph_,
-    node_services_,
-    node_logging_,
-    node_clock_,
-    node_parameters_
-  )),
+      node_base_,
+      node_topics_,
+      node_graph_,
+      node_services_,
+      node_logging_,
+      node_clock_,
+      node_parameters_
+    )),
```

Reply:
@vinnamkim Can you please update to fix the formatting?

Reply:
@Karsten1987, @mjeronimo Sorry for repeated miss formatting. I updated commit and checked ament_uncrustify, ament_cppcheck, and colcon test.